README.md

```diff
@@ -274,77 +274,147 @@ Response (Un-authenticated)
 }
 ```
 
-### feeds/friends
-When the app opens, it makes a call to `api/feed/friends`
+### feeds/friends-v1
+When the app opens, it makes a call to `api/feed/friends-v1`
 
 Below is an example, with PII removed. 
 ```text
-[
-  {
-    "id": "<>>",
-    "notificationID": "<>>",
-    "ownerID": "<>>",
-    "userName": "<>>",
+{
+  "userPosts": {
     "user": {
-      "id": "<>",
-      "username": "<>"
+      "id": "[REDACTED]",
+      "username": "[REDACTED]",
+      "profilePicture": {
+        "url": "https://cdn.bereal.network/Photos/[REDACTED]/profile/[REDACTED]",
+        "width": 999,
+        "height": 999
+      }
     },
-    "mediaType": "late",
     "region": "us-central",
-    "bucket": "storage.bere.al",
-    "photoURL": "https://storage.bere.al/Photos/<>/bereal/f822aa7a-3b0c-495f-b5bd-a4250a465083-1659997174.jpg",
-    "imageWidth": 1500,
-    "imageHeight": 2000,
-    "secondaryPhotoURL": "https://storage.bere.al/Photos/<>/bereal/f822aa7a-3b0c-495f-b5bd-a4250a465083-1659997174-secondary.jpg",
-    "secondaryImageHeight": 2000,
-    "secondaryImageWidth": 1500,
-    "members": [
-      "<>"
-    ],
-    "lateInSeconds": 67,
-    "isPublic": false,
-    "location": {
-      "_latitude": <>,
-      "_longitude": -<>
-    },
-    "retakeCounter": 0,
-    "creationDate": {
-      "_seconds": 1659997238,
-      "_nanoseconds": 684000000
-    },
-    "updatedAt": 1660002563356,
-    "takenAt": {
-      "_seconds": 1659997174,
-      "_nanoseconds": 0
-    },
-    "comment": [],
-    "realMojis": [
+    "momentId": "L1O8S_Tbph2fpEeuoSWcv",
+    "posts": [
       {
-        "id": "-j-<>>",
-        "uid": "<>>",
-        "userName": "<>>",
-        "user": {
-          "id": "<>>",
-          "username": "<>>",
-          "profilePicture": {
-            "height": 1000,
-            "width": 1000,
-            "url": "https://storage.bere.al/cdn-cgi/image/width=200,height=200/Photos/<>/profile/<>-1658600169-profile-picture.jpg"
-          }
+        "id": "-PGe3UUH4wsDg0ttqc7G-",
+        "visibility": ["friends"],
+        "primary": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+          "width": 2000,
+          "height": 1500
         },
-        "emoji": "😍",
-        "type": "heartEyes",
-        "uri": "https://storage.bere.al/Photos/<>/realmoji/<>-realmoji-heartEyes-1659046828.jpg",
-        "date": {
-          "_seconds": 1660002563,
-          "_nanoseconds": 350000000
-        }
+        "secondary": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+          "width": 2000,
+          "height": 1500
+        },
+        "caption": "I love my tortilla blanky 🥰",
```

Comment:
I don't know the context, but I can get on board with this

Reply:
Context: https://cdn.bereal.network/Photos/zRHCCjGLIZZ7EnliQwSMZ6zacj23/post/GkZr_Wx0jeCJ7uFu.webp